.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,42 @@
+name: Push to Pypi
+
+on:
+  release:
+    types: [released, prereleased]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install --no-cache-dir -e .[complete]
```

Comment:
This is really rather odd, I won't install a package _and_ build the wheel. The wheel build should be correct regardless of what is installed (use PEP 518 to control requirements).